docs/samples/explanation/income/Makefile

```diff
@@ -5,7 +5,11 @@ model.joblib features.joblib category_map.joblib:
 .PHONY: train
 train: model.joblib features.joblib category_map.joblib
 
+.PHONY: run_predictor
+run_predictor: model.joblib
+	python -m sklearnserver --model_dir ./  --model_name income --protocol seldon.http
+
 .PHONY: clean
 clean:
 	rm -f *.joblib
-
+	rm -f *.dill
```

Comment:
afaik we don't support .dill extension as of now?

Reply:
This is specific to the AlibiExplainer server which assumes the explainer is trained and saved using dill. This is because at present Alibi uses lambdas at present internally so pickle can't be used to save the explainer.